src/components/BytebaseBanner.vue

```diff
@@ -1,18 +1,20 @@
 <template>
-  <div class="w-full px-3 max-w-2xl mx-auto flex flex-col justify-center items-center">
+  <div
+    class="w-full px-3 max-w-2xl mx-auto flex flex-col justify-center items-center"
```

Comment:
I am not sure why it causes format change here since you are the only one working on this.

Have you recently changed your formatting style in your IDE?

Reply:
I just seen that you had changed this file before with https://github.com/bytebase/star-history/commit/bad68b08deb113c0181cfed8b3d2cd320091553c
I will do later: make the format configs(eslint and prettier) same to `bytebase`.